src/main/java/frc/robot/subsystems/AutonSubsytem.java

```diff
@@ -139,20 +109,89 @@ private Command getAutonSequence(){
                return 
                     new SequentialCommandGroup(
                         resetOdometry("Forward"),
-                        followTrajectory("Forward")
+                        swerveSubsystem.followTrajectory("Forward", getTrajectory("Forward"))
                     );
             case BACKWARD:
                 return
                     new SequentialCommandGroup(
                         resetOdometry("Backward"),
                         delay(3),
-                        followTrajectory("Backward")
+                        swerveSubsystem.followTrajectory("Backward", getTrajectory("Backward"))
                     );
             case FORWARD180:
                 return
                     new SequentialCommandGroup(
                         resetOdometry("Forward180"),
-                        followTrajectory("Forward180")
+                        swerveSubsystem.followTrajectory("Forward180", getTrajectory("Forward180"))
+                    );
+            case CURVE:
+                return
+                    new SequentialCommandGroup(
+                        resetOdometry("Curve"),
+                        swerveSubsystem.followTrajectory("Curve", getTrajectory("Curve"))
+                    );
+            case ONESCORECHARGETOP:
+                return
+                    new SequentialCommandGroup(
+                        resetOdometry("1ScoreChargeTop"),
+                        swerveSubsystem.followTrajectory("1ScoreChargeTop", getTrajectory("1ScoreChargeTop"))
+                    );
+            case ONESCORECHARGEMID:
+                return
+                    new SequentialCommandGroup(
+                        resetOdometry("1ScoreChargeMid"),
+                        swerveSubsystem.followTrajectory("1ScoreChargeMid", getTrajectory("1ScoreChargeMid"))
+                    );
+            case ONESCORECHARGELOW:
+                return
+                    new SequentialCommandGroup(
+                        resetOdometry("1ScoreChargeLow"),
+                        swerveSubsystem.followTrajectory("1ScoreChargeLow", getTrajectory("1ScoreChargeLow"))
+                    );
+            case TWOSCORETOP:
+                return
+                    new SequentialCommandGroup(
+                        resetOdometry("2ScoreTop1"),
+                        swerveSubsystem.followTrajectory("2ScoreTop1", getTrajectory("2ScoreTop1")),
+                        swerveSubsystem.followTrajectory("2ScoreTop2", getTrajectory("2ScoreTop2"))
```

Comment:
For trajectories with multiple segments, does the resetOdometry method need to be called between them?

Reply:
No, reset odometry is called only once